ZenPacks/zenoss/LinuxMonitor/modeler/plugins/zenoss/cmd/linux/lvm.py

```diff
@@ -354,7 +367,15 @@ def makeLVMap(self, columns):
             lv_om.origin = columns['origin']
             lv_om.relname = 'snapshotVolumes'
             lv_om.modname = 'ZenPacks.zenoss.LinuxMonitor.SnapshotVolume'
+        elif columns['lv_metadata_size']:
+            lv_om.metadatasize = columns['lv_metadata_size']
+            lv_om.relname = 'thinPools'
+            lv_om.modname = 'ZenPacks.zenoss.LinuxMonitor.ThinPool'
         else:
+            if columns['pool_lv']:
+                lv_om.set_thinPool = 'lv-{}'.format(self.prepId(columns['vg_name'])+'_'+self.prepId(columns['pool_lv']))
+            else:
+                lv_om.set_thinPool = ''
             lv_om.relname = 'logicalVolumes'
             lv_om.modname = 'ZenPacks.zenoss.LinuxMonitor.LogicalVolume'
         return lv_om
```

Comment:
I'm getting a traceback on Debian9:

2018-03-13 15:45:11,473 ERROR zen.ZenModeler: : Traceback (most recent call last):
  File "/opt/zenoss/Products/ZenHub/PBDaemon.py", line 101, in inner
    return callable(*args, **kw)
  File "/opt/zenoss/Products/ZenHub/services/ModelerService.py", line 163, in remote_applyDataMaps
    if adm._applyDataMap(device, map, commit=False):
  File "/opt/zenoss/Products/DataCollector/ApplyDataMap.py", line 126, in _applyDataMap
    return self._applyDataMapImpl(device, datamap)
  File "/opt/zenoss/Products/DataCollector/ApplyDataMap.py", line 176, in _applyDataMapImpl
    tobj = device.getObjByPath(datamap.compname)
  File "/opt/zenoss/Products/ZenModel/ZenModelBase.py", line 625, in getObjByPath
    return getObjByPath(self, path)
  File "/opt/zenoss/Products/ZenUtils/Utils.py", line 301, in getObjByPath
    next=obj[name]
  File "/opt/zenoss/lib/python2.7/site-packages/OFS/ObjectManager.py", line 777, in __getitem__
    raise KeyError, key
KeyError: 'volumeGroups'